### PR TITLE
Fixes #13545: Require Postgresql 9.2 for rudder server

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/13545

### CODE REVIEW

File: rudder-webapp/SPECS/rudder-webapp.spec [hunk 1]
```diff
@@ -95,7 +95,7 @@ BuildArch: noarch
 Requires: rudder-techniques = %{real_epoch}:%{real_version}, rudder-server-relay = %{real_epoch}:%{real_version}, ncf-api-virtualenv = %{real_epoch}:%{real_version}, %{apache}, %{apache_tools}, git-core, rsync, openssl, rudder-jetty = %{real_epoch}:%{real_version}, %{ldap_clients}
 
 # We need the PostgreSQL client utilities so that we can run database checks and upgrades (rudder-upgrade, in particular)
-Requires: postgresql >= 8.4
+Requires: postgresql >= 9.2
```
Comment (peckpeck) [inline rudder-webapp/SPECS/rudder-webapp.spec]: isn't there an equivalent on debian ?

File: rudder-webapp/SOURCES/rudder-upgrade [hunk 2]
```diff
@@ -502,25 +502,22 @@ upgrade_database() {
   # Now check the fileFormat in the eventlog
   upgrade_eventlog
 
-  # - All versions: Check for the PostgreSQL version (>= 8.4)
+  # - All versions: Check for the PostgreSQL version
   STEP="Check for the current PosgresSQL version"
 
-  # If installing on SLES11 and PostgreSQL 8.3 was used
+  # If coming from an old SLES11 install
```
Comment (peckpeck) [inline rudder-webapp/SOURCES/rudder-upgrade]: technically you cannot install rudder 5 on sles11

Comment (amousset) [reply rudder-webapp/SOURCES/rudder-upgrade]: I don't exactly know how migration from SLES 11 to SLES12 works, but I guess the old Postgresql could still be used (like on debian for example).

File: rudder-webapp/SPECS/rudder-webapp.spec (hunk 1 above)
Comment (amousset) [reply rudder-webapp/SPECS/rudder-webapp.spec]: It was (surprisingly) already 9.2...

**Pull request merged**